highcharts-api/highcharts/7-connecting-column-edges/main.js

```diff
@@ -0,0 +1,54 @@
+const generateRandomData = (length) => Array.from({ length }, () => Math.random() * 14);
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'column',
+
+        events: {
+            render() {
+                const chart = this;
+                console.log(this);
+
+                let zIndex = 3;
+                
+                for(data of chart.series) {
+                    let prevCorner = null;
+                    let lineIndex = 0;
+                    zIndex++;
+
+                    if(!data.lines) data.lines = [];
+
+                    for(point of data.points) {
+                        const x = chart.plotLeft + point.shapeArgs.x,
+                            y = chart.plotTop + point.shapeArgs.y + 1;
+
+                        if(prevCorner?.length) {
```

Comment:
In Highcharts, we do not try not to use optional chaining because the code transpiled by TypeScript looks very unreadable later

```suggestion
                        if(prevCorner && prevCorner.length) {
```